### ISSUE TITLE
How much we should leavrage existing RFCs ? 

### ISSUE THREAD

**Issue by raszuk** [billwuqin/Green-Energy-Network#5]
Specifically https://datatracker.ietf.org/doc/html/rfc8348 ? I do not see this mentioned in the charter. 

I see draft-ietf-ccamp-network-inventory-yang-02 builds upon this RFC.  So I would recommend to reference this draft in the charter.

**Comment by billwuqin** [comment]
Yes, Good point and will see how to integrate it in the next version.

**Comment by jariarkko** [comment]
Agree
